app/views/sessions/unauthenticated.haml

```diff
@@ -1,20 +1,17 @@
-%h1 Welcome to the website for Special Transportation.
+%h1 Welcome to the #{t 'department.name'} website.
 %h2 You are not allowed on this page.
 %p
   Some possible scenarios:
   %ul
     %li
       You are a current or future passenger. In this case, you might be
       interested in our
-      = link_to 'brochure.', brochure_passengers_path
-    %li
-      You are an employee of UMass Transit, but we haven't added you to our
-      database yet. In this case, please talk to an ST Supervisor or contact
-      IT.
+      = succeed '.' do
+        = link_to 'brochure', brochure_passengers_path
     %li
-      You are an employee of UMass Transit and should be able to log in.
-      If this is the case, please talk to an ST Supervisor to get it sorted
-      out.
```

Comment:
I couldn't see much meaningful difference between these two scenarios. They're both, "you work here, but don't have access, ask staff."